executioncontext/projectview/src/test/java/org/jetbrains/bsp/bazel/projectview/model/ProjectViewBuilderTest.kt

```diff
@@ -230,7 +271,14 @@ class ProjectViewBuilderTest {
                     javaPath = ProjectViewJavaPathSection(Paths.get("imported/path/to/java")),
                     buildFlags = ProjectViewBuildFlagsSection(
                         listOf("--build_flag1.1=value1.1", "--build_flag1.2=value1.2")
-                    )
+                    ),
+                    directories = ProjectViewDirectoriesSection(
+                            listOf(
+                                    Path("included_dir1.1"), Path("included_dir1.2")
+                            ),
+                            listOf(Path("excluded_dir1.1"))
+                    ),
+                    deriveTargetsFlag = ProjectViewDeriveTargetsFlagSection(true)
```

Comment:
just to test returning the last defined values

```suggestion
                    deriveTargetsFlag = ProjectViewDeriveTargetsFlagSection(false)
```

Reply:
Good catch. Thanks